blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,75 @@
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/dvteam_activated
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories:
+- news
+---
+
+## Starting a new adventure with RgSoC and Living Style Guide from coding newbies to professional front-end developers in 11 months.
+
+### Better Together
+November 2017, we started as complete coding newbies, with as little coding knowledge as HTML can be. JavaScript, the word alone, felt like a entire new universe. We can’t even believe how much we have learned in the past 8 months! Coming from a non technical background and being able to code in so little time has been quite a journey. Yet, it made us realize we have definitely unknown abilities and a real passion for programming.
+
+
+We started with a selected group of 20.000 out of 200.000 applications in Europe, Africa and Middle East region. Earning a full Google Scholarship for Front End Web Developer at Udacity was the fruit of this three months of dedication, hard work, discipline, cooperation with peers coders and taking actively part in a huge social community, in order to stand out and win our spot in the top 2.000 students (out of 20.000). We are now about to finish the last part: React.
+
+### RgSoC Coaches & Supervisor
+During the Google Scholarship, we met several times in real life and decided to apply for RgSoc to be able to keep practicing our coding skills in summer. As we were new in the dev community, we reached out to our Udacity community managers who introduced us to Amanda Cavallaro. She helped us find our first two current RgSoc-coaches: 
+
+*Coach 1: Tobias (React developer)* 
+
+*Coach 2: Tetiana (Ruby developer)*
+
+Skyping with Tobias and Tetiana, gave us the confidence they would have our backs! They not only seem very nice and accessible, but moreover have the attitude and experience needed to guide us through the summer.  
+
+A few weeks later we met our third mentor through the post system in RGSoC.We absolutely loved his passion and diverse knowledge of programming so we both agreed to have him on board!
+
+*Coach 3: Raounak (Full-stack Ruby developer)*
+
+The last RgSoc-coach to join, we met during a recently peer lab, that Violeta organized for Woman Who Code. We really liked her spirits and clarity so just a week before starting the SoC we invited her to be part of it. 
+
+*Coach 4: Cristina (full stack developer)*
+
+We consider ourselves very lucky! As not only do we have gathered a great team of mentors, but also an interesting supervisor Lucas Pinto who has even organized Ruby conferences! 
+
+### Living Style Guide
+The Open Source project we have been selected for is Living Style Guide, which helps developers to create front-end style guides easily with SaSS and Markdown. The project is written by Nico Hagenburger in Ruby. It needs to be rewritten to and tested for a Vanilla Javascript version, which will be our main tasks, beside improving the documentation. 
+
+Summer and future Goals

```

Comment:
This looks like some title, so may be it makes sense to add some style or colon here. Add one blank line too. At the moments it is just the same sentence which looks weird.